daemon/daemon_main.go

```diff
@@ -1085,6 +1088,17 @@ func initEnv(cmd *cobra.Command) {
 		}
 	}
 
+	if option.Config.EnableDSR {
+		switch {
+		case !option.Config.EnableNodePort:
+			log.Fatal("DSR can be enabled only for NodePort BPF (--enable-node-port)")
+		case option.Config.EnableDSR && option.Config.EnableIPv6:
+			log.Fatal("DSR cannot be used with IPv6")
```

Comment:
PR for IPv6 support will also come before 1.7 freeze, right?

Reply:
Yep.